index.bs

```diff
@@ -203,8 +230,27 @@ spec: mediaqueries-4; urlPrefix: https://drafts.csswg.org/mediaqueries-4/#
       </p>
 
       <p>
-        A <dfn>valid VideoConfiguration contentType</dfn> MUST include a media
-        type, subtype and full codecs description.
+        To check if a {{VideoConfiguration}} <var>configuration</var> is a
+        <dfn>valid video configuration</dfn>, the following steps MUST be run:
+        <ol>
+          <li>
+            If <var>configuration</var> is <code>null</code>, return
+            <code>true</code> and abort these steps.
+          </li>
+          <li>
+            If <var>configuration</var>'s {{VideoConfiguration/contentType}} is
```

Comment:
Should there not be some other step in some other part of the spec that checks if the content type is supported (not just syntactically valid) which could make these checks unnecessary? Treating "foo/nonsense" the same as "video/x-matroska" or some other unsupported video container seems fine?